### PR TITLE
Fixed typos, error, and made documentation more descriptive.

### PR DESCRIPTION
**Pull request by astromariarosa**
There were typos in the original documentation, and an error in the expected output of the factorial function. Additionally, the documentation did not have helpful explanations.

### CODE REVIEW

**Comment by lavanyanemani1996** [comment]
Reviewing this

**Comment by RubenArjona** [comment]
Reviewing this as well

File: Marinelli/template/README.md [hunk 1]
```diff
@@ -1,13 +1,12 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
+This code has two functions. The first function gives the factorial of a given number.
 
     >>> from simple_functions import factorial
     >>> factorial(10)
-    9
+    3628800
```
Comment (lavanyanemani1996) [inline Marinelli/template/README.md:5]: Function factorial works

File: Marinelli/template/README.md [hunk 2]
```diff
@@ -1,13 +1,12 @@
-Ok so I guess you are reading this cuz you wanna use my code. There are some
-functions that do stuf and thats:
```
Comment (thomaitsiftsi) [inline Marinelli/template/README.md:2]: I wonder why you deleted all this?

**Review by thomaitsiftsi — COMMENTED** [review]
It looks ok to me!

File: Marinelli/template/README.md (hunk 2 above)
Comment (astromariarosa) [reply Marinelli/template/README.md:2]: I thought it wasn't very helpful, so I replaced it with the next line.

Comment (astromariarosa) [reply Marinelli/template/README.md:2]: @thomaitsiftsi Hey, can you see this notification?

Comment (thomaitsiftsi) [reply Marinelli/template/README.md:2]: Yes, at last! Thank you for the explanation.

**Comment by astrofrog** [comment]
Thanks for opening the pull request and for the review(s)!  We will merge this shortly.  Below I have provided a comment about the *process* of the pull request and review. But note that we are not conducting a review of the code itself, because the exercise was about the overall process not the details of the code.

* @thomaitsiftsi @lavanyanemani1996 - you did not select "approve" or "request changes" when you submitted the review. Without that a maintainer isn't sure if your review should be taken as a complete review or just general comments.

**Pull request merged**